lib/sycamore/sycamore/tests/integration/query/execution/test_sycamore_query.py

```diff
@@ -19,9 +19,9 @@ def test_simple(self, query_integration_test_index: str, codegen_mode: bool):
             schema,
             natural_language_response=True,
         )
-        query_id, result = client.run_plan(plan, codegen_mode=codegen_mode)
-        assert isinstance(result, str)
-        assert len(result) > 0
+        result = client.run_plan(plan, codegen_mode=codegen_mode)
+        assert isinstance(result.result, str)
```

Comment:
seems like this is failing

Reply:
It was passing for me but the test is still flaky